examples/meta/generator/translate.py

```diff
@@ -412,6 +412,12 @@ def getIncludePathForClass(self, type_):
                                      uniqueCandidates[0]))
             return uniqueCandidates[0]
 
+        # hack since ctags cannot find include paths for preprocessor
```

Comment:
@sorig @lisitsyn any ideas how to make ctags read function names constructed using the double `##` in preprocessor, see [here](https://github.com/shogun-toolbox/shogun/pull/4494/files#diff-8454fda42e54fd37dd5549559efce515R52) ?